app/frontend/src/App.vue

```diff
@@ -17,10 +24,23 @@ import Footer from './components/Footer';
 
 export default {
   name: 'app',
+  data() {
+    return {
+      snack: false
+    };
+  },
   components: {
     Header,
     NavigationBar,
     Footer
+  },
+  mounted() {
+    setTimeout(() => {
+      this.snack = !this.$router;
+      if (!this.$router && (this.$keycloak && !this.$keycloak.ready)) {
```

Comment:
All AND conditions, don't need the brackets

Reply:
Fixed